tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -235,8 +233,7 @@ stages:
 
       - template: "templates/use-android-ndk.yml"
 
-      - script: /bin/bash tools/ci_build/github/android/setup_gradle_wrapper.sh $(pwd)
-        displayName: Setup gradle wrapper to use gradle 6.8.3
+      - template: templates/set-up-gradle-wrapper-step.yml
```

Comment:
My understanding on the android related changes are just refactoring, am I right?

Reply:
yes, build definition changes are refactoring. if we need to fix this urgently (before someone from onnxruntime-es takes a look), only the changes in java/build.gradle are necessary.